src/utils.js

```diff
@@ -1015,3 +1015,33 @@ export const getSalePriceMultiplier = (completedAchievements = {}) => {
 
   return salePriceMultiplier
 }
+
+/**
+ * @param {Array} an array of objects each containing a `weight` property
```

Comment:
I just noticed this because my editor appropriately highlights the variable name, but for this JSDoc syntax to be correct, this line needs to begin with `@param {Array} weightedOptions [description...]`.